### PR TITLE
XFCE Survey (December 2023)

### PR DESCRIPTION
**Pull request by Fearyncess**
Topic Description
-----------------

This topic surveys for available XFCE component and application updates as of December 2023.

Package(s) Affected
-------------------

### Modified
- xfce4-screensaver: 4.18.2
- xfce4-taskmanager: 1.5.6
- garcon: 4.18.1
- xfce4-appfinder: 4.18.1
- tumbler: 4.18.2
- libxfce4ui: 4.18.4
- xfce4-session: 1:4.18.3
- xfdesktop: 4.18.1
- orage: 4.18.0
- xfce4-notifyd: 0.9.3
- xfce4-pulseaudio-plugin: 0.4.8
- xfce4-calculator-plugin: 0.7.2
- xfce4-cpugraph-plugin: 1.2.10
- xfce4-netload-plugin: 1.4.1
- thunar: 4.18.8
- xfce4-weather-plugin: 0.11.1
- xfce4-mailwatch-plugin: 1.3.1
- xfce4-power-manager: 4.18.3
- mousepad: 0.6.1
- xfce4-notes-plugin: 1.10.0
- xfce4-panel: 1:4.18.5
- xfce4-datetime-plugin: 0.8.3
- xfce4-genmon-plugin: 4.2.0
- ristretto: 0.13.1
- xfce4-whiskermenu-plugin: 2.8.2
- xfce4-panel-profiles: 1.0.14
- xfce4-battery-plugin: 1.1.5
- xfce4-dev-tools: 4.18.1
- xfce4-fsguard-plugin: 1.1.3
- parole: 4.18.1
- xfconf: 4.18.3
- xfce4-terminal: 1.1.1
- gigolo: 0.5.3
- xfce4-screenshooter: 1.10.4
- xfce4-mpc-plugin: 0.5.3
- xfburn: 0.7.0
- catfish: 4.18.0
- xfce4-timer-plugin: 1.7.2
- xfce4-settings: 4.18.4
- xfce4-clipman-plugin: 1.6.5

### Introduced
- gtk-layer-shell: 0.8.1

Security Update?
----------------

No

Build Order
-----------

```
#buildit gtk-layer-shell xfce4-dev-tools xfconf libxfce4ui garcon xfce4-panel xfce4-panel-profiles thunar xfdesktop xfce4-notifyd xfce4-power-manager xfce4-settings xfce4-session mousepad orage tumbler xfce4-appfinder parole ristretto xfburn gigolo catfish xfce4-screenshooter xfce4-taskmanager xfce4-terminal xfce4-calculator-plugin xfce4-pulseaudio-plugin xfce4-screensaver xfce4-battery-plugin xfce4-clipman-plugin xfce4-cpugraph-plugin xfce4-datetime-plugin xfce4-fsguard-plugin xfce4-genmon-plugin xfce4-mailwatch-plugin xfce4-mpc-plugin xfce4-netload-plugin xfce4-notes-plugin xfce4-timer-plugin xfce4-weather-plugin xfce4-whiskermenu-plugin
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

**Second Architectures**

- [x] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by Fearyncess** [comment]
@aosc-buildit-bot build amd64

**Comment by eatradish** [comment]
> @aosc-buildit-bot build amd64

为什么你只编译一个架构？

**Comment by Fearyncess** [comment]
> > @aosc-buildit-bot build amd64
> 
> 为什么你只编译一个架构？

懒狗，等amd64走通，依赖没问题再说。

**Review by Fearyncess — COMMENTED** [review]
@aosc-buildit-bot build amd64

**Review by Fearyncess — COMMENTED** [review]
@aosc-buildit-bot build amd64

**Comment by eatradish** [comment]
?

**Comment by eatradish** [comment]
@aosc-buildit-bot build amd64

**Review by Fearyncess — COMMENTED** [review]
@aosc-buildit-bot build amd64

File: desktop-xfce/parole/spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #   are still not in a newer release. So the `parole` package should use a git
 #   snapshot instead of a release tarball.
 
-VER=4.16.0+git20221230
+VER=4.18.1
 SRCS="git::commit=389d6a72b08f90c15a5a193c7da8f9df25575ec6::https://gitlab.xfce.org/apps/parole"
```
Comment (MingcongBai) [inline desktop-xfce/parole/spec]: You probably haven't updated this yet.

File: desktop-xfce/xfce4-screensaver/spec [hunk 2]
```diff
@@ -1,4 +1,4 @@
-VER=4.16.0+git20221004
+VER=4.18.2
 SRCS="git::commit=785a456fd843c574def813fdb616162812631ebf::https://gitlab.xfce.org/apps/xfce4-screensaver"
```
Comment (MingcongBai) [inline desktop-xfce/xfce4-screensaver/spec]: Update this actually.

File: runtime-desktop/gtk-layer-shell/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PKGNAME="gtk-layer-shell"
+PKGSEC=libs
+PKGDES="A library to write GTK applications that use Layer Shell."
+PKGDEP="gtk-3 wayland wayland-protocols gobject-introspection vala"
+ABTYPE=meson
```
Comment (MingcongBai) [inline runtime-desktop/gtk-layer-shell/autobuild/defines]: Missing newline.

File: runtime-desktop/gtk-layer-shell/spec [hunk 4]
```diff
@@ -0,0 +1,4 @@
+VER=0.8.1
+SRCS="tbl::https://github.com/wmww/gtk-layer-shell/archive/refs/tags/v$VER.tar.gz"
+CHKSUMS="sha256::c329fac878a1731cb23ad7365f2f2a35e4ab26e72b4e69458e06afd825baad48"
+CHKUPDATE="anitya::id=198568"
```
Comment (MingcongBai) [inline runtime-desktop/gtk-layer-shell/spec]: Missing newline.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — DISMISSED** [review]

File: runtime-desktop/gtk-layer-shell/autobuild/defines [hunk 5]
```diff
@@ -0,0 +1,5 @@
+PKGNAME="gtk-layer-shell"
+PKGSEC=libs
+PKGDES="A library to write GTK applications that use Layer Shell."
```
Comment (MingcongBai) [inline runtime-desktop/gtk-layer-shell/autobuild/defines]: Remove the period point at the end.

File: runtime-desktop/gtk-layer-shell/autobuild/defines [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+PKGDEP="gtk-3 wayland wayland-protocols gobject-introspection vala"
```
Comment (MingcongBai) [inline runtime-desktop/gtk-layer-shell/autobuild/defines]: Are you sure that all these are runtime dependencies?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: runtime-desktop/gtk-layer-shell/autobuild/defines (hunk 5 above)
Comment (Fearyncess) [reply runtime-desktop/gtk-layer-shell/autobuild/defines]: i have sent the dot back home.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**